sthree/registry.go

```diff
@@ -41,10 +41,12 @@ func newBucketRegistry() *bucketRegistry {
 
 func (r *bucketRegistry) init() {
 	auth, err := aws.EnvAuth()
-	grip.CatchDebug(err)
 	if err != nil {
-		auth, err = aws.SharedAuth()
-		grip.CatchDebug(err)
+		var err2 error
```

Comment:
I don't think you need err2 here. You can just reassign to err, since you don't handle the case where err is not nil but err2 is nil.